edvart/data_types.py

```diff
@@ -4,7 +4,7 @@
 import pandas as pd
 
 try:
-    import pyarrow  # pylint: disable=unused-import
+    import arrow  # pylint: disable=unused-import
```

Comment:
Where did you find this out? In the [documentation](https://arrow.apache.org/cookbook/py/io.html) I see it's imported as `pyarrow`. Could you add link to some resource in PR description?

Reply:
Oh, I just realized what was wrong 🤦. It was because of my IDE complaining with `pyarrow` and not complaining with `arrow`. Which was because I installed `arrow` instead of `pyarrow` due to the bad `pyproject.toml` config :grin:.

Reply:
I see 😁